main/modes/games/cGrove/Sparring/cg_Spar.c

```diff
@@ -86,7 +83,11 @@ void cg_initSpar(cGrove_t* grove)
     // Initialize battle record
     sparLoadBattleRecords();
 
+    // Play BGM
+    globalMidiPlayerPlaySong(&cg->spar.sparBGM, MIDI_BGM);
```

Comment:
Is this meant to loop? Does it need `globalMidiPlayerGet(MIDI_BGM)->loop = true;`, or is that guaranteed to be set elsewhere?

Reply:
I set the player to loop when grabbed it. Unless I need to set it to loop every time the song is changed.

Reply:
I think that's OK, but if you haven't double checked that, double check that.